contracts/deposit-contract/depositContract.v.py

```diff
@@ -15,7 +15,7 @@
 deposit_count: uint256
```

Comment:
Can you set deposit_count, full_deposit_count, skip_chainstart_delay to public while you are here?